CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/genericjsonio/JsonIODialog.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2010-2020 Australian Signals Directorate
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.functionality.genericjsonio;
+
+import au.gov.asd.tac.constellation.utilities.datastructure.Tuple;
+import au.gov.asd.tac.constellation.utilities.gui.DraggableCell;
+//import au.gov.asd.tac.constellation.visual.DraggableCell;
+import java.util.Optional;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.event.ActionEvent;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Button;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.ListView;
+import javafx.scene.control.TextInputDialog;
+
+/**
+ * Displays a generic dialog window that can allow the user to select a Json
+ * preference from a list
+ *
+ * @author formalhaut69
+ */
+public class JsonIODialog {
```

Comment:
Since this doesn't appear to rely on graphs or anything else high-level, could you please move it into Core Utilities.

Reply:
Movel this - and JsonIO (below) into CoreUtilties,